README.md

```diff
@@ -19,61 +19,61 @@ Serenata de Amor is an open project using artificial intelligence for social con
 
 ### Who
 
-We are a group of people who believes in _power to the people_ motto. We are also part of the _Data Science for Civic Innovation Programme_ from [Open Knowledge Brasil](http://br.okfn.org).
+We are a group of people who believes in _power to the people_. We are also part of the _Data Science for Civic Innovation Programme_ from [Open Knowledge Brasil](http://br.okfn.org).
```

Comment:
Why remove “motto” when this is actually a motto?

Reply:
Since most people are already aware that "power to the people" is a motto, there is not really a need to say that it is.

Reply:
> Since most people are already aware that "power to the people" is a motto

We'll teach them : )